pkg/minishift/openshift/version/openshift_versions.go

```diff
@@ -91,19 +92,23 @@ func PrintDownStreamVersions(output io.Writer, minSupportedVersion string) {
 }
 
 func PrintUpStreamVersions(output io.Writer, minSupportedVersion string) {
-	resp, err := getResponseBody("https://registry.hub.docker.com/v2/repositories/openshift/origin/tags/")
+	dockerRegistryUrl := "https://registry.hub.docker.com/v2/repositories/openshift/origin/tags/?page_size=1024"
```

Comment:
> I am wondering whether we should v1 of the API - https://registry.hub.docker.com/v1/repositories/openshift/origin/tags. This seems to return in a more condensed format what we want and it is proper v1 API syntax.

V1 url has the same issue of some empty fields and gave below error, I un-comment the error checking code. 

```
$ minishift openshift version list
*json.UnmarshalTypeError
json: cannot unmarshal array into Go value of type version.ImageTags
&json.UnmarshalTypeError{Value:"array", Type:(*reflect.rtype)(0xa8b980), Offset:1}
```
Also V1 URL is not working at all with the current code as it is getting the above error when we try to parse it.